site/guide/documentation-section-statuses.qmd

```diff
@@ -0,0 +1,34 @@
+---
+title: "Assign documentation section statuses"
+date: last-modified
+---
+
+Assign a completion status to individual sections of your model documentation that will be reflected in your Document Overview.
+
+## Prerequisites
+- You hold the `Developer` role 
+- The model is already [registered in the model inventory](register-models-in-model-inventory.qmd)
+
+## Steps
+
+1. [Log in to the ValidMind UI]({{< var vm_url >}}).
+
+2. In the left sidebar, click [**Model Inventory**]({{< var vm_url >}}/model-inventory).
+
+3. Select a model by clicking on it or find your model by applying a filter or searching for it.
+
+4. In the left sidebar that appears for your model, click **Documentation**.
+
+   You can now jump to any section of the model documentation by selecting the relevant section you would like to view.
+
+5. Next to the heading of each section, **you can assign a Status from a drop-down menu:** 
+    - In Progress
+    - Done
+
+   ::: {.callout-tip appearance="minimal"}
+    The documentation completion Status summary will then be updated on your Document Overview.
```

Comment:
```suggestion
    The documentation completion status summary will then be updated on your Document Overview page.
```

AFAIK, "Status" is not a proper noun and "Document Overview" is the name of a page, so it might help to just include that. 

(Not a strong opinion, but I find that page name a bit confusing as it's not shown when you first click on the overview; the bread crumb just refers to "Documentation" the name "Document Overview" is visible only when you are in subsections.)